src/Microsoft.Azure.WebJobs.Host/Scale/IScalerProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    /// <summary>
+    /// Interface for providing scalers.
+    /// </summary>
+    public interface IScalerProvider
```

Comment:
This ends up being required if we want to allow extensions to use DI properly w/o having to new instances up manually, which breaks DI flow. TBD on exact name